access/gitlab/webhook_server.go

```diff
@@ -93,12 +94,18 @@ func (s *WebhookServer) processWebhook(rw http.ResponseWriter, r *http.Request,
 		return
 	}
 
-	body, err := ioutil.ReadAll(r.Body)
+	// as per https://docs.gitlab.com/ee/user/gitlab_com/index.html#webhooks,
+	// the webhook payload size is limited to 25MB
+	body, err := ioutil.ReadAll(io.LimitReader(r.Body, 30*1024*1024))
```

Comment:
Mismatch between comment and code - we're giving them an extra 5MB over what the doc says. Are we allowing a little padding for some overhead?

Might also be an idea to pull it out into a constant as its used in a couple of places here.

Reply:
I was leaving some overhead room but the webhook payloads shouldn't get anywhere near that size to begin with, I'll just make it adhere to the declared limits instead.